spec/features/web_helper.rb

```diff
@@ -0,0 +1,24 @@
+def sign_up
+  visit('/sign_up')
+  fill_in('name', with: 'Alaan')
+  fill_in('username', with: 'Alaanzr')
+  fill_in('email', with: 'Alaanzr@gmail.com')
+  fill_in('password', with: 'password')
+  click_button('Register')
+end
+
+def partial_sign_up
+  fill_in('name', with: 'Alaan')
+  click_button('Register')
+end
+
+def sign_out
+  click_button('Sign out')
+end
+
+def sign_in(name, username, email, password)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `name`. If it's necessary, use `_` or `_name` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)<br>Lint/UnusedMethodArgument: Unused method argument - `email`. If it's necessary, use `_` or `_email` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)